examples/tutorials/flow-deploy/distribute-flow-as-executable-app/main.py

```diff
@@ -82,7 +83,7 @@ def list_iter_render_message(message_items):
                 st.markdown(f"`{json_dumps(message_items)},`")
 
         def dict_iter_render_message(message_items):
-            if is_multimedia_dict(message_items):
+            if BaseMultimediaProcessor.is_multimedia_dict(message_items):
```

Comment:
We updated the example flows. Does this mean that we made some breaking changes to the users?

Reply:
Confirmed with @YingChen1996 , it does have a small chance of affecting customers, Ying will follow up.